pkg/core/options.go

```diff
@@ -76,6 +78,13 @@ func WithMetrics(enabled bool) Option {
 	}
 }
 
+// WithLabel toggles metrics for the core package and its dependencies (e.g. cafs)
```

Comment:
doc string probably results from a copy/paste

Reply:
will attempt to make a note of this in the forthcoming change at layers after `storage`